intervals/main_objs.py

```diff
@@ -888,6 +896,146 @@ def distance(self, df=None, n=3):
     def getDistance(self, df=None, n=3):
         return self.distance(df, n)
 
+    # July 2022 helper for flexed entries
+    def _flexed_sum(self, item, head_flex):
```

Comment:
There are two versions of _flexed_sum that appear to be identical. We should definitely just have one.

Reply:
Done